### ISSUE TITLE
How to see the full backtrace from rake task

### ISSUE THREAD

**Issue by justin808** [bbatsov/rubocop#1840]
I only get this error from the rake task, not the command line.

Note, I'm not overriding the default value of `Style/WordArray`.

```
Inspecting 296 files
...................................................An error occurred while Style/WordArray cop was inspecting /Users/justin/clients/madrone/investor-dashboard/app/models/exporters/export_managers.rb.
To see the complete backtrace run rubocop -d.

Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.30.1 (using Parser 2.2.2.1, running on ruby 2.2.1 x86_64-darwin14)
RuboCop failed!
```

I tried this:

```
  RuboCop::RakeTask.new(:rubocop) do |task|
    task.options = ["-d"]
  end
```

But that does not give me anything interesting.

Is this a potential issue if the command line and rake task differ?

**Comment by JaniJegoroff** [comment]
For me your rake task looks about correct. I'm using below rake task without problems:
```
RuboCop::RakeTask.new(:rubocop) do |t|
  t.options = %w(-d)
end
```
What is your `rake` gem version?

**Comment by justin808** [comment]
rake (10.4.2)

**Comment by JaniJegoroff** [comment]
I'm using the same `rake` version.

**Comment by alexdowad** [comment]
This makes RuboCop print backtraces for me:

```ruby
RuboCop::RakeTask.new(:rubocop) do |t|
  t.options = %w(-d)
end
```

**Comment by alexdowad** [comment]
Same thing when running `v0.30.1`.

**Comment by alexdowad** [comment]
This can be closed, I think.
